rios/applier.py

```diff
@@ -897,10 +897,11 @@ def opensAsRaster(filename):
         ds = gdal.Open(filename)
```

Comment:
I guess we can remove the `if hasattr(gdal, 'GetUseExceptions')` thing and always assume that it is available? All the other `GetUseExceptions` calls here assume it does...